src/Analyzer/FileParser.php

```diff
@@ -26,7 +25,7 @@ public function __construct(NodeTraverser $traverser, FileVisitor $fileVisitor,
 
         $lexer = new Emulative([
             'usedAttributes' => ['comments', 'startLine', 'endLine', 'startTokenPos', 'endTokenPos'],
-             'phpVersion' => self::PHP_VERSION,
+             'phpVersion' => phpversion('tidy'),
```

Comment:
tidy is not needed

Reply:
for the coding standard, we need to use it.

Reply:
You are right, I have removed the extension @micheleorselli